src/main/java/org/cryptimeleon/craco/sig/sps/SPSPublicParameters.java

```diff
@@ -1,80 +0,0 @@
-package org.cryptimeleon.craco.sig.sps;
-
-import org.cryptimeleon.craco.common.PublicParameters;
-import org.cryptimeleon.math.serialization.Representation;
-import org.cryptimeleon.math.serialization.annotations.ReprUtil;
-import org.cryptimeleon.math.serialization.annotations.Represented;
-import org.cryptimeleon.math.structures.groups.GroupElement;
-import org.cryptimeleon.math.structures.groups.elliptic.BilinearGroup;
-import org.cryptimeleon.math.structures.groups.elliptic.BilinearMap;
-import org.cryptimeleon.math.structures.rings.zn.Zp;
-
-import java.util.Objects;
-
-/**
- * An interface containing generic components shared by many SPS schemes
- * i.e. a bilinear group for evauating pairing product equations and the associated
- * group generators
- * */
-public abstract class SPSPublicParameters implements PublicParameters {
```

Comment:
why is this class gone?

Reply:
I f I remember correctly, when I originally wrote the scheme, this class caused problems with the representation tests - which is why I gave up on it.
Now that I have a little more experience with the library though, I managed to get it working. The newest commit in the branch should have `AGHO11PublicParameters extends SPSPublicParameters` now, while passing all existing test cases.

Reply:
Update: I messed up with the changes above, which I have corrected now. Now the StandaloneRepresentation tests no longer fail for `SPSPublicParameters`. However, this means that the class is no longer abstract (because that clashed with the RepresentationTester, which picked up the constructor and tried to instantiate it, causing failure.)